examples/simple_example/variables.tf

```diff
@@ -34,3 +34,14 @@ variable "bucket_policy_only" {
   type        = map(string)
 }
 
+variable "lifecycle_rules" {
```

Comment:
Since we're hardcoding this now, can we remove the variable?

Reply:
@kopachevsky To be clear, I don't want *any* variables for lifecycle_rules in either examples or fixtures, it can just be hardcoded in the example.

Reply:
did it in example

Reply:
But you still have a variable in the example and fixtures. Please remove the variables, as I've said a number of times.

Reply:
sorry, my mistake